doc/src/fix_wall_region.txt

```diff
@@ -10,19 +10,24 @@ fix wall/region command :h3
 
 [Syntax:]
 
-fix ID group-ID wall/region region-ID style epsilon sigma cutoff :pre
-
-ID, group-ID are documented in "fix"_fix.html command
-wall/region = style name of this fix command
-region-ID = region whose boundary will act as wall
-style = {lj93} or {lj126} or {lj1043} or {colloid} or {harmonic}
-epsilon = strength factor for wall-particle interaction (energy or energy/distance^2 units)
-sigma = size factor for wall-particle interaction (distance units)
-cutoff = distance from wall at which wall-particle interaction is cut off (distance units) :ul
+fix ID group-ID wall/region region-ID style args ... cutoff :pre
+
+ID, group-ID are documented in "fix"_fix.html command :ulb,l
+wall/region = style name of this fix command :l
+region-ID = region whose boundary will act as wall :l
+style = {lj93} or {lj126} or {lj1043} or {morse} or {colloid} or {harmonic} :l
+args = :l
```

Comment:
I think the 3 Morse params should be the same names and order as in the Morse pair style,
assuming the formula is essentially the same.  The first section under args should be for the lj and other styles.  Then add a section for style = morse.

Reply:
@sjplimp how about this updated variant?

Reply:
all non-morse styles use the same number of parameters. for harmonic, the sigma parameter is ignored and epsilon is used as force constant (hence the two different units mentioned).
i don't want to change this, since this has been done like this for *ages*. also on the regular wall styles. and breaking backward compatibility would be difficult to justify.